neofetch

```diff
@@ -1507,203 +1514,205 @@ get_uptime() {
-
-            case $os-$kernel_name in
-                BSD-FreeBSD|BSD-DragonFly)
-                    has pkg && tot pkg info
-                ;;
-
-                BSD-*)
-                    has pkg && dir /var/db/pkg/*
-
-                    ((packages == 0)) &&
-                        has pkg && tot pkg list
-                ;;
-            esac
-
-            # List these last as they accompany regular package managers.
-            has flatpak && tot flatpak list
-            has spm     && tot spm list -i
-            has puyo    && dir ~/.puyo/installed
-
-            # Snap hangs if the command is run without the daemon running.
-            # Only run snap if the daemon is also running.
-            has snap && ps -e | grep -qFm 1 snapd >/dev/null && \
-            pkgs_h=1 tot snap list && ((packages-=1))
-
-            # This is the only standard location for appimages.
-            # See: https://github.com/AppImage/AppImageKit/wiki
-            manager=appimage && has appimaged && dir ~/.local/bin/*.appimage
-        ;;
-
-        "Mac OS X"|"macOS"|MINIX)
-            has port  && pkgs_h=1 tot port installed && ((packages-=1))
-            has brew  && dir "$(brew --cellar)/* $(brew --caskroom)/*"
-            has pkgin && tot pkgin list
-            has dpkg  && tot dpkg-query -f '.\n' -W
-
-            has nix-store && {
-                nix-user-pkgs() {
-                    nix-store -qR ~/.nix-profile
-                    nix-store -qR /etc/profiles/per-user/"$USER"
-                }
-                manager=nix-system && tot nix-store -qR /run/current-system/sw
-                manager=nix-user   && tot nix-user-pkgs
-            }
-        ;;
-
-        AIX|FreeMiNT)
-            has lslpp && ((packages+=$(lslpp -J -l -q | grep -cv '^#')))
-            has rpm   && tot rpm -qa
-        ;;
-
-        Windows)
-            case $kernel_name in
-                CYGWIN*) has cygcheck && tot cygcheck -cd ;;
-                MSYS*)   has pacman   && tot pacman -Qq --color never ;;
-            esac
-
-            # Scoop environment throws errors if `tot scoop list` is used
-            has scoop && pkgs_h=1 dir ~/scoop/apps/* && ((packages-=1))
-
-            # Count chocolatey packages.
-            [[ -d /cygdrive/c/ProgramData/chocolatey/lib ]] && \
-                dir /cygdrive/c/ProgramData/chocolatey/lib/*
-        ;;
-
-        Haiku)
-            has pkgman && dir /boot/system/package-links/*
-            packages=${packages/pkgman/depot}
-        ;;
-
-        IRIX)
-            manager=swpkg
-            pkgs_h=3 tot versions -b && ((packages-=3))
-        ;;
-    esac
-
-    if ((packages == 0)); then
-        unset packages
-
-    elif [[ $package_managers == on ]]; then
-        printf -v packages '%s, ' "${managers[@]}"
-        packages=${packages%,*}
-
-    elif [[ $package_managers == tiny ]]; then
-        packages+=" (${manager_string%,*})"
+    if [[ $show_package_total == "on" ]]; then
```

Comment:
It would be much better to return early if `$show_package_total == "off"` rather than re-indent the entire function